src/export-ops-json-payloads.ts

```diff
@@ -0,0 +1,105 @@
+import { join } from 'path';
+import fs from 'fs';
+import * as child_process from 'child_process';
+import { readOperationsRepository } from './utils/read-operations';
+import { writeJsonFile } from './utils/write-operations';
+import { sanitiseFilename } from './utils/filesystem';
+
+const main = () => {
+  const commitHash = child_process.execSync('git rev-parse HEAD').toString().trim();
+
+  const basePath = join(__dirname, '../json-exports');
+  fs.rmdirSync(basePath, { recursive: true });
+  fs.mkdirSync(basePath);
+
+  const opsFull = readOperationsRepository();
+
+  writeJsonFile(join(basePath, 'operations.json'), opsFull);
+
+  Object.entries(opsFull).map(([key, value]) => {
+    writeJsonFile(join(basePath, `${key}.json`), value);
+  });
+
+  // Grabbed from https://github.com/api3dao/operations/blob/eadde827d78ddbe1858c5fe7f5b00f953d77dc14/src/utils/normalization.ts#L105
```

Comment:
We'll probably want this in a single place I assume. Since we are in a hurry to merge this PR we might want to track that refactor in a separate issue.

Reply:
That was an old commit. Portions of this script were copied from the old documentation export. This is principally to support Warren/API3 Docs - so this is _the_ place; it isn't done anywhere else AFAIK.